apps/data-pipeline/library-traffic-scraper/wrangler.toml

```diff
@@ -0,0 +1,19 @@
+name = "library-traffic-scraper"
+main = "src/handler.ts"
+compatibility_date = "2024-12-30"
+compatibility_flags = ["nodejs_compat"]
+workers_dev = false
+minify = true
+
+[triggers]
+crons = ["*/15 * * * *"]
+
+[observability]
+enabled = true
+
+[[hyperdrive]]
+binding = "DB"
+id = "1c4c82477bfa4114a48c63d4dc838e73"
+
+[env.development.vars]
+DB_URL = "postgres://postgres:postgres@localhost:5432/anteater_api"
```

Comment:
we never set this var in any other package in the import layer, and it's not good practice to make assumptions about people's dev configurations which are intentionally allowed to vary widely